src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
+
+.header {
+  width: 100%;
+  min-width: 1024px;
+  display: flex;
+  background-color: white;
+}
+
+.logo {
+  margin: auto;
+  margin-left: 50px;
+}
+
+.nav {
+  margin: auto;
+  width: 100%;
+  margin-right: 50px;
+}
+
+.list {
+  display: flex;
+  justify-content: end;
   margin: 0;
 }
+
+.nav-link {
+  list-style-type: none;
+  margin-left: 20px;
+}
```

Comment:
The 'nav-link' class has a left margin specified. According to the checklist, it's better to be consistent with vertical margins. In most cases, it's recommended to add only top or only bottom margins, but not both.